Classes/Command/DatabaseDocCommand.php

```diff
@@ -0,0 +1,292 @@
+<?php
```

Comment:
This didn't work.

```
Uncaught TYPO3 Exception Class 'TYPO3\CMS\Extbase\Reflection\DocCommentParser' not found
thrown in file /var/www/extensions/dlf/Classes/Command/DatabaseDocCommand.php
in line 188
```

I don't like this approach anyway. The TYPO3 scheduler / CLI task should never write into `typo3conf/*`. During development there might be exceptions like we use lfeditor to edit the XLIFF files. But lfeditor is in this case an external tool. This CLI job is part of Kitodo.Presentation and is equal to all other CLI jobs. Every user could add a Scheduler task to render the Documents.rst file in a read-only directory.

Hope you get the point. All this should go to `Build/`.

Reply:
Oh, apparently the `DocCommentParser` has been removed in TYPO3 v10. For now I integrated it into the command if you would like to try it out.

If it's just about the command being run by other users, we could check for `TYPO3_CONTEXT=Development`.

But yes, it would be nicer if this could go into `Build/` and be called directly from the extension. The issue was that I didn't know how to do this, because I'm relying on TYPO3, e.g. for getting the classmap. I'll think about it.